packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -129,7 +129,9 @@ export class LoadTestDataStoreModel {
 		// If we did not create the data store above, load it by getting its url.
 		if (gcDataStore === undefined) {
 			const gcDataStoreId = root.get(gcDataStoreIdKey);
-			const response = await containerRuntime.request({ url: `/${gcDataStoreId}` });
+			const response = await (containerRuntime as ContainerRuntime).resolveHandle({
```

Comment:
These tests should not be using resolveHandle. I created a task to re-visit this AB#6334

Reply:
Not blocking for this PR of course.